### ISSUE TITLE
[Report] Error in starting times

### ISSUE THREAD

**Issue by yourkiwifruit** [crashinvaders/gdx-texture-packer-gui#88]
com.badlogic.gdx.utils.GdxRuntimeException: Unable to invoke method: init of type: com.crashinvaders.texturepackergui.controllers.extensionmodules.ExtensionModuleRepositoryService@3ef47b51 with parameters: [] at com.github.czyzby.autumn.context.impl.method.MethodInvocation.invoke(MethodInvocation.java:50)

<details><summary>Application log</summary>

```
Version: 4.8.2
OS: Windows 10 10.0 amd64
JRE: 14.0.2 Oracle Corporation
External log file: C:\Users\ZhuXiaoBing\.gdxtexturepackergui\logs\2010171147.log
com.badlogic.gdx.utils.GdxRuntimeException: Unable to invoke method: init of type: com.crashinvaders.texturepackergui.controllers.extensionmodules.ExtensionModuleRepositoryService@3ef47b51 with parameters: []
	at com.github.czyzby.autumn.context.impl.method.MethodInvocation.invoke(MethodInvocation.java:50)
	at com.github.czyzby.autumn.processor.impl.InitiateAnnotationProcessor.doAfterScanning(InitiateAnnotationProcessor.java:42)
	at com.github.czyzby.autumn.context.ContextInitializer.invokeProcessorActionsAfterInitiation(ContextInitializer.java:351)
	at com.github.czyzby.autumn.context.ContextInitializer.initiate(ContextInitializer.java:294)
	at com.crashinvaders.texturepackergui.App.initiateContext(App.java:100)
	at com.crashinvaders.texturepackergui.App.create(App.java:85)
	at com.badlogic.gdx.backends.lwjgl.LwjglCanvas.create(LwjglCanvas.java:210)
	at com.badlogic.gdx.backends.lwjgl.LwjglCanvas$1.addNotify(LwjglCanvas.java:100)
	at java.desktop/java.awt.Container.addNotify(Container.java:2800)
	at java.desktop/javax.swing.JComponent.addNotify(JComponent.java:4783)
	at java.desktop/java.awt.Container.addNotify(Container.java:2800)
	at java.desktop/javax.swing.JComponent.addNotify(JComponent.java:4783)
	at java.desktop/java.awt.Container.addNotify(Container.java:2800)
	at java.desktop/javax.swing.JComponent.addNotify(JComponent.java:4783)
	at java.desktop/java.awt.Container.addNotify(Container.java:2800)
	at java.desktop/javax.swing.JComponent.addNotify(JComponent.java:4783)
	at java.desktop/javax.swing.JRootPane.addNotify(JRootPane.java:729)
	at java.desktop/java.awt.Container.addNotify(Container.java:2800)
	at java.desktop/java.awt.Window.addNotify(Window.java:778)
	at java.desktop/java.awt.Frame.addNotify(Frame.java:490)
	at java.desktop/java.awt.Window.show(Window.java:1040)
	at java.desktop/java.awt.Component.show(Component.java:1717)
	at java.desktop/java.awt.Component.setVisible(Component.java:1664)
	at java.desktop/java.awt.Window.setVisible(Window.java:1023)
	at com.crashinvaders.texturepackergui.desktop.launchers.awt.AwtFrameLauncher$1.run(AwtFrameLauncher.java:50)
	at java.desktop/java.awt.event.InvocationEvent.dispatch(InvocationEvent.java:316)
	at java.desktop/java.awt.EventQueue.dispatchEventImpl(EventQueue.java:770)
	at java.desktop/java.awt.EventQueue$4.run(EventQueue.java:721)
	at java.desktop/java.awt.EventQueue$4.run(EventQueue.java:715)
	at java.base/java.security.AccessController.doPrivileged(AccessController.java:391)
	at java.base/java.security.ProtectionDomain$JavaSecurityAccessImpl.doIntersectionPrivilege(ProtectionDomain.java:85)
	at java.desktop/java.awt.EventQueue.dispatchEvent(EventQueue.java:740)
	at java.desktop/java.awt.EventDispatchThread.pumpOneEventForFilters(EventDispatchThread.java:203)
	at java.desktop/java.awt.EventDispatchThread.pumpEventsForFilter(EventDispatchThread.java:124)
	at java.desktop/java.awt.EventDispatchThread.pumpEventsForHierarchy(EventDispatchThread.java:113)
	at java.desktop/java.awt.EventDispatchThread.pumpEvents(EventDispatchThread.java:109)
	at java.desktop/java.awt.EventDispatchThread.pumpEvents(EventDispatchThread.java:101)
	at java.desktop/java.awt.EventDispatchThread.run(EventDispatchThread.java:90)
Caused by: com.badlogic.gdx.utils.reflect.ReflectionException: Exception occurred in method: init
	at com.badlogic.gdx.utils.reflect.Method.invoke(Method.java:114)
	at com.github.czyzby.kiwi.util.gdx.reflection.Reflection.invokeMethod(Reflection.java:130)
	at com.github.czyzby.autumn.context.impl.method.MethodInvocation.invoke(MethodInvocation.java:47)
	... 37 more
Caused by: java.lang.reflect.InvocationTargetException
	at java.base/jdk.internal.reflect.NativeMethodAccessorImpl.invoke0(Native Method)
	at java.base/jdk.internal.reflect.NativeMethodAccessorImpl.invoke(NativeMethodAccessorImpl.java:62)
	at java.base/jdk.internal.reflect.DelegatingMethodAccessorImpl.invoke(DelegatingMethodAccessorImpl.java:43)
	at java.base/java.lang.reflect.Method.invoke(Method.java:564)
	at com.badlogic.gdx.utils.reflect.Method.invoke(Method.java:108)
	... 39 more
Caused by: com.badlogic.gdx.utils.SerializationException: Error reading file: .gdxtexturepackergui/modules/repo_cache.json
	at com.badlogic.gdx.utils.Json.fromJson(Json.java:726)
	at com.crashinvaders.texturepackergui.controllers.extensionmodules.ExtensionModuleRepositoryService.init(ExtensionModuleRepositoryService.java:46)
	... 44 more
Caused by: com.badlogic.gdx.utils.SerializationException: Error parsing file: .gdxtexturepackergui/modules/repo_cache.json
	at com.badlogic.gdx.utils.JsonReader.parse(JsonReader.java:78)
	at com.badlogic.gdx.utils.Json.fromJson(Json.java:724)
	... 45 more
Caused by: com.badlogic.gdx.utils.SerializationException: Error parsing JSON on line 3 near: 
<!DOCTYPE html>
*ERROR*<!--

Hello future GitHubber! I bet you're here to remove those 
	at com.badlogic.gdx.utils.JsonReader.parse(JsonReader.java:550)
	at com.badlogic.gdx.utils.JsonReader.parse(JsonReader.java:56)
	at com.badlogic.gdx.utils.JsonReader.parse(JsonReader.java:76)
	... 46 more
```

</details>

**Comment by metaphore** [comment]
Thanks for the report. A new version is coming next week, it most likely will fix this one.

**Comment by yourkiwifruit** [comment]
That's great

**Comment by metaphore** [comment]
The issue was fixed in the [4.9.0](https://github.com/crashinvaders/gdx-texture-packer-gui/releases/tag/4.9.0) release. Feel free to reopen if the problem persists.
